tests/acceptance/features/bootstrap/TagContext.php

```diff
@@ -179,4 +179,21 @@ public function userRemovesTagsFromResourceOfTheSpace(string $user, string $file
 		);
 		$this->featureContext->setResponse($response);
 	}
+
+	/**
+	 * @Given  /^user "([^"]*)" has removed the following tags for (folder|file)\s?"([^"]*)" of space "([^"]*)":$/
+	 *
+	 * @param string $user
+	 * @param string $fileOrFolder   (file|folder)
+	 * @param string $resource
+	 * @param string $space
+	 * @param TableNode $table
+	 *
+	 * @return void
+	 * @throws Exception
+	 */
+	public function userHAsRemovedTheFollowingTagsForFileOfSpace(string $user, string $fileOrFolder, string $resource, string $space, TableNode $table):void {
+		$this->userRemovesTagsFromResourceOfTheSpace($user, $fileOrFolder, $resource, $space, $table);
+		$this->featureContext->theHttpStatusCodeShouldBe(200);
```

Comment:
if we don't have to check any response in Given then code lgtm

Reply:
The check is for that the request to remove tag was successful. No other response checks is done.

Reply:
we don't get any response body